### DISCUSSION TITLE
email invites not working -

### DISCUSSION THREAD

**Discussion by MarkhamLee** [supabase/supabase#27760] [Questions]
I'm having an issue where my manager is trying to send me an invite to Supabase and they don't work. 

Replication steps: 
1) He sends me an invite 
2) I receive the email, click the link
3) The login screen has a message: "There was an error requesting details for this invitation. (No authorization token was found)", so if I sign to accept it just created an organization for me, but didn't add me to the one for my job. I've tried other email accounts, browsers (Chrome, Firefox) and even used my phone as a hot spot to connect in case my firewall was removing the token somehow. Nothing worked. 

Anyone got any ideas?

![Screenshot from 2024-07-02 14-59-02](https://github.com/supabase/supabase/assets/16950894/8a26d2fc-2999-4a1a-89bd-edc54acfee43)

**Comment by stevengrimaldo** [comment]
Having the same issue just now.
